test/test_utils.py

```diff
@@ -492,11 +493,26 @@ def try_check_onnx_broadcast(dims1, dims2, expect_broadcast, expect_fail):
         try_check_onnx_broadcast(dims1, dims2, True, False)
 
 
+# Errors will still be raised and reported
+@contextlib.contextmanager
+def suppress_stderr():
+    original = sys.stderr
+    sys.stderr = open(os.devnull, 'w')
+    yield
+    sys.stderr = original
+
+
 class TestHub(TestCase):
     @classmethod
     @skipIfNoTorchVision
     def setUpClass(cls):
-        cls.resnet18_pretrained = models.__dict__['resnet18'](pretrained=True).state_dict()
+        # The current torchvision code does not provide a way to disable tqdm
+        # progress bar, leading this download printing a huge number of lines
+        # in CI.
+        # TODO: remove this context manager when torchvision provides a way.
+        #       See pytorch/torchvision#862
```

Comment:
This prints more than 40% of the CI log:
![image](https://user-images.githubusercontent.com/5674597/56558961-4b8f4800-656e-11e9-92bd-54042253856a.png)


torchvision issue: https://github.com/pytorch/vision/issues/862